components/accreditation/AccreditationProjectCard.vue

```diff
@@ -0,0 +1,29 @@
+<script lang="ts" setup>
+import { asImageSrc } from '@prismicio/helpers'
```

Comment:
Oh sorry, I forgot to change it, I have a snippet that does this by default 😅

Since Vue 3 with the composition API and script setup, the Vue team has been using script first because it lets you know what props, variables, etc. you're gonna define and use in the template without having to scroll through the template. I do prefer it (but do not have a strong opinion about it) because, since I did the switch, I noticed I was learning faster what the component did than the other way around.